manifests/init.pp

```diff
@@ -77,6 +77,8 @@
   $config_dir                       = $kafka::params::config_dir,
   Stdlib::Absolutepath $bin_dir     = $kafka::params::bin_dir,
   $log_dir                          = $kafka::params::log_dir,
+  $manage_group                     = $kafka::params::manage_group,
```

Comment:
huh the Boolean type isn't passing tests, nor syntax check on my side.  I've not strictly typed variables before, but using this document (http://www.example42.com/2015/09/09/puppet4-examples-data-types/) as a reference I seem to have the syntax correct.  Can you provide any direction for this?